posts/2023-12-31-foss-engines-overview.md

```diff
@@ -0,0 +1,95 @@
+
+### Stride 
+* Written in C#, Supported Languages: C#, F#, and Visual Basic
+* Platforms:
+    * Development: Windows, Linux (Code only)
+    * Target: Windows, Linux, Android, iOS
+    
+Stride began life as Xenko (and before that, Paradox): Silicon Studio's premium game engine. After several years of providing Stride to the public through a subscription-based model, Silicon Studio released the engine's source code and editor freely to the community under the MIT license. Among the major FOSS game engines available, it is unique because Silicon Studio completely wrote it in C# from top to bottom. There is no delineation between the language used for the core engine and the language you would write with day-to-day while working on the game. It becomes much easier to override or change any inherent engine behavior when coding in the same language. With that said, the code-only version of Stride supports any language that is part of the .NET family (C#, F#, and Visual Basic), providing some flexibility in language choice. 
+
+Stride offers a pure .NET C# experience that includes many of the advantages inherent to the framework, like hot reloading. Because the engine closely follows the .NET update schedule, you regularly get the most modern and up-to-date implementation of C#. Anything you might expect to do with a C# development environment is possible here. You can seamlessly incorporate almost any C# library or tool available through NuGet, GitHub, or other platforms into Stride, enhancing your workflow. Stride is modular enough that sections of Stride are available as standalone NuGet packages. The engine provides the ability to tailor-make your C# game development experience.
+
+* Graphics Compositor
+
+### Godot
+* Written in C++, Supported Languages: GDScript, C#
+* Platforms:
+    * Development: Windows, Linux, Mac
+    * Target: Windows, Linux, Mac, iOS, Android, WASM, HTML5
+
+Among all the modern FOSS game engines available to date, Godot has the largest and most active community. As the drama around Unity and Unreal has folded, you have more than likely heard mentions of Godot more than a few occasions. It is not without merit, as Godot encourages developers to shape the engine around their needs. Coming in at only 40 MB, the engine includes a lightweight, multi-platform editor capable of running on any major operating system (Windows, Mac, and Linux). In fact, you can even use a web-based version, albeit with some constraints. Godot can meet developers on whatever platform works best for them.
```

Comment:
which drama around unreal?

Reply:
Specifically, the layoffs affecting parts of the Unreal Engine development team back in September, which signified some long-term changes to Epic's vision for UE, including placing a growing emphasis on UEFN.

That said, this was a much lower profile flash point compared to the Unity runtime fee, so I'm open to removing the reference to Unreal.

Reply:
Removed the reference to Unreal, as that wasn't as notable of an event.